addons/l10n_vn/data/account_tax_report_data.xml

```diff
@@ -56,12 +56,23 @@
                                     </record>
                                 </field>
                             </record>
+                            <record id="account_tax_report_line_04_01_01_vn" model="account.report.line">
+                                <field name="name">VAT Exemption on purchase of goods and services</field>
+                                <field name="code">VAT_EXEMPTION_ON_PURCHASE_OF_GOODS_AND_SERVICES</field>
```

Comment:
@aliyatastemirova Hi, do you mind if i export the term here because i export it at my local seem we have missed a lot of term ?

Reply:
nice catch! I can see that quite a few translations are missing. If it's not too difficult for you, maybe we could add all the unrelated missing terms in the existing translation files in a separate commit (but within this PR)? If you happen to have translations for them, it would be amazing :pray:

Reply:
@aliyatastemirova Done, i have added translation which is needed, can you take a look if you have time please